qutebrowser/config/configdata.yml

```diff
@@ -3675,6 +3685,8 @@ bindings.default:
       <Ctrl-X>: prompt-open-download
       <Ctrl-P>: prompt-open-download --pdfjs
       <Shift-Tab>: prompt-item-focus prev
+      <Ctrl-N>: prompt-history prev
+      <Ctrl-P>: prompt-history next
```

Comment:
Argh, this is going to conflict with the existing `Ctrl-P` binding for `prompt-open-download --pdfjs`. I don't have a good idea for how to solve this right now. I'll get back to it, but it'll likely take me a while.

Reply:
Perhaps there is another binding? I think rebinding the pdfjs binding wouldn't be good because it would mess with a lot of people's workflows.

Perhaps the binding of the history for the command prompt could also be changed along with it so it is consistent? I think most people just use the up and down keys anyways so that might be less intrusive.